proxy/src/http.rs

```diff
@@ -21,6 +21,23 @@ mod source;
 mod transaction;
 mod user;
 
+/// Helper to combine the multiple filters together with Filter::or, possibly boxing the types in
```

Comment:
```suggestion
/// Helper to combine the multiple filters together with Filter::or, boxing the types in
```
?

Reply:
Whoops, I got too excited about the PR and forgot to apply this change : |